lib/charms/loki_k8s/v0/log_proxy.py

```diff
@@ -81,6 +81,24 @@ def __init__(self, *args):
 the charm that implements the `loki_push_api` interface, the library will inject a
 Pebble layer that runs Promtail in the workload container to send logs.
 
+The default behaivour is that Promtail binary that is injected into the workload container
+is downloaded from internet, but in some scenarios workload containers cannot reach internet.
+In this situation the following section should be added to `resources` section in metadata.yaml
```

Comment:
```suggestion
The default behaivour is that Promtail binary that is injected into the workload container
is downloaded from internet, but in some scenarios workload containers cannot reach internet.
To enable people operating in these situations, the following section should be added to the 
`resources` section of your charm's `metadata.yaml`, which will allow operators to attach a binary
at runtime should they need, rather than rely on fetching from the internet.
```